src/Uploadcare/Uploader.php

```diff
@@ -54,13 +54,18 @@ public function status($token)
    * @return File|string
    * @throws \Exception
    */
-  public function fromUrl($url, $check_status = true, $timeout = 1, $max_attempts = 5)
+  public function fromUrl($url, $fileName = null, $store = true, $check_status = true, $timeout = 1, $max_attempts = 5)
```

Comment:
1. don't change order of arguments
2. default for $store should be 'auto', as this will break uploads for projects with disabled autostore